superset/assets/src/components/TableSelector.jsx

```diff
@@ -277,10 +274,10 @@ export default class TableSelector extends React.PureComponent {
         name="select-table"
         ref="selectTable"
         isLoading={this.state.tableLoading}
+        ignoreAccents={false}
         placeholder={t('Select table or type table name')}
         autosize={false}
         onChange={this.changeTable}
-        filterOptions={this.state.filterOptions}
```

Comment:
there is a unit test for filterOptions prop. are you sure to remove it?

Reply:
oops, fixed!